packages/cosmo/package.py

```diff
@@ -48,9 +48,9 @@ class Cosmo(MakefilePackage):
     apngit   = 'git@github.com:MeteoSwiss-APN/cosmo.git'
     maintainers = ['elsagermann']
 
-    version('master', branch='master')
-    version('dev-build', branch='master')
-    version('mch', git='git@github.com:MeteoSwiss-APN/cosmo.git', branch='mch')
+    version('master', branch='master', get_full_repo=True)
+    version('dev-build', branch='master', get_full_repo=True)
+    version('mch', git='git@github.com:MeteoSwiss-APN/cosmo.git', branch='mch', get_full_repo=True)
     version('gt2', git='git@github.com:havogt/cosmo.git', branch='gt2')
```

Comment:
We don't do it for the gt2 version, because it's only supposed to exist temporarily until the gt2.0 upgrade has been merged after which it will be removed again?

Reply:
yes exaclty!